src/main/resources/db/migration/V1_2__create_table_oppfolgingstilfelle_arbeidstaker.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE OPPFOLGINGSTILFELLE_ARBEIDSTAKER
+(
+    id                                SERIAL PRIMARY KEY,
+    uuid                              VARCHAR(50) NOT NULL UNIQUE,
```

Comment:
Kan vi ha `VARCHAR(36)`, eller evt. `CHAR(36)` her, siden en uuid alltid er 36 tegn lang?

Reply:
Du mener 40?(36 + 4 stk bindestrek).
Tror også det skal være en egen uuid-greie for dette i PSQl, så kan evt. titte mer på det.

Reply:
Har nå oppdatert begge kolonnene til `VARCHAR(36)`.